Tests/SonarScanner.MSBuild.Tasks.UnitTests/GetAnalyzerSettingsTests.cs

```diff
@@ -359,7 +359,7 @@ private GetAnalyzerSettings Execute_ConfigExists(string sonarQubeVersion, string
                 SonarQubeHostUrl = "https://localhost:9000",
                 ServerSettings = new AnalysisProperties
                 {
-                    // Server settings should be ignored
+                    // Server settings should be ignored. "true" value should break existing tests.
```

Comment:
shouldn't this commit be on  #986?

Reply:
The entire commit. `Review update 2`, as well as `Review update 3` bellow should have been on #986. I've accidentally push them to a wrong branch. But they'll get delivered.